### PR TITLE
all: migrate ssi packages to ecs@mappings

### PR DESCRIPTION
**Pull request by efd6**
## Proposed commit message

```
Migration performed using ecs-update.

  go run github.com/andrewkroh/go-examples/ecs-update@014b35dfe4c9832b51e7c909a39a48257d6a005d \
    -ecs-version=8.11.0 \
    -ecs-git-ref=v8.11.0 \
    -fields-yml-drop-ecs \
    -kibana-version=^8.13.0 \
    -drop-import-mappings \
    -pr=10135 \
    -owner=elastic/security-service-integrations \
    packages/*
```

## Checklist

- [ ] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [ ] I have verified that all data streams collect metrics or logs.
- [ ] I have added an entry to my package's `changelog.yml` file.
- [ ] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).

## Author's Checklist

- [ ] 

## Related issues

- Depends #10136
- Depends #10134
- Depends #10186

### CODE REVIEW

File: packages/1password/changelog.yml [hunk 1]
```diff
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: "1.29.0"
+  changes:
+    - description: Update the kibana constraint to ^8.13.0. Modified the field definitions to remove ECS fields where possible.
```
Comment (andrewkroh) [inline packages/1password/changelog.yml:4]: From a user perspective, do you think this second sentence is confusing? Maybe we should mention the `ecs@mappings` component template in some way. Like

> Removed ECS field definitions that have been made redundant by the `ecs@mappings` component template.

File: packages/cel/changelog.yml [hunk 2]
```diff
@@ -1,3 +1,8 @@
+- version: "1.12.0"
+  changes:
+    - description: Update the kibana constraint to ^8.13.0.
```
Comment (andrewkroh) [inline packages/cel/changelog.yml:3]: I think there might be a bug with ecs-update and input type packages. cel has some `external: ecs` fields that were not removed during this update. It probably is related to this bit of incomplete work https://github.com/andrewkroh/go-fleetpkg/issues/5.

File: packages/1password/changelog.yml (hunk 1 above)
Comment (efd6) [reply packages/1password/changelog.yml:4]: That seems reasonable.

File: packages/cel/changelog.yml (hunk 2 above)
Comment (efd6) [reply packages/cel/changelog.yml:3]: Ack

File: packages/1password/changelog.yml (hunk 1 above)
Comment (efd6) [reply packages/1password/changelog.yml:4]: andrewkroh/go-examples#36

**Comment by chrisberkhout** [comment]
An issue to be aware of for integrations with transforms: https://github.com/elastic/elastic-package/issues/1641

**Comment by chrisberkhout** [comment]
> ~An issue to be aware of for integrations with transforms: [elastic/elastic-package#1641](https://github.com/elastic/elastic-package/issues/1641)~
> 
> Correction: actually not an issue since we're not importing them now. They should just be there, presumably for transform destination indexes as well as data streams.

Update: Transform destination indexes don't get the ECS dynamic templates that data streams get. This PR doesn't remove the manual definitions in transforms, so it still works (checked for ti_opencti).

File: packages/ti_opencti/data_stream/indicator/fields/ecs.yml [hunk 3]
```diff
@@ -39,174 +35,12 @@
     - x509-certificate
     - unknown
     - port
-# Additional file hash algorithms
```
[the remark this answers is not in the document]
Comment (efd6) [reply packages/ti_opencti/data_stream/indicator/fields/ecs.yml:42]: This is most likely an issue in goccy/go-yaml. The AST there does not treat comments as first class entities; they get attached to other nodes, and I've seen that the way this is done is less than ideal in many cases. I think we can add these back in another PR. WDYT?

[the remark this answers is not in the document]
Comment (chrisberkhout) [reply packages/ti_opencti/data_stream/indicator/fields/ecs.yml:42]: Sure. In the cases I saw those comments were nice to have but not critical. They'll still be in git history. I wouldn't bring them back unless it's easy.

**Review by chrisberkhout — APPROVED** [review]

File: packages/akamai/changelog.yml
Comment (Jaos1992) [inline packages/akamai/changelog.yml:1]: > __

**Pull request merged**